src/data/mystery-encounters/encounters/safari-zone-encounter.ts

```diff
@@ -304,14 +304,14 @@ async function throwBait(scene: BattleScene, pokemon: EnemyPokemon): Promise<boo
   bait.setOrigin(0.5, 0.625);
   scene.field.add(bait);
 
-  scene.playSound("pb_throw");
-
   return new Promise(resolve => {
     scene.trainer.setTexture(`trainer_${scene.gameData.gender === PlayerGender.FEMALE ? "f" : "m"}_back_pb`);
     scene.time.delayedCall(512, () => {
+      scene.playSound("pb_throw");
+
       // Trainer throw frames
       scene.trainer.setFrame("2");
-      scene.time.delayedCall(256, () => {
+      scene.time.delayedCall(184, () => {
```

Comment:
magic number? It's being reused

Reply:
just for specific animation timing, pretty arbitrary